HW03/zayats_aliaksei_hw03_t_04.rb

```diff
@@ -0,0 +1,15 @@
+def symbol_count(string)
+  letters = 0
+  digits = 0
+  digit_pattern = /[0-9]/
+  letter_pattern = /[a-zA-Z]/
+  string.split("").each do |symbol|
```

Comment:
`chars` do the same 😉 
```suggestion
  string.chars.each do |symbol|
```